quicktests/overlaying/tests/realistic/hockey.js

```diff
@@ -0,0 +1,71 @@
+function makeData() {
+  "use strict";  
+
+}
+
+function run(svg, data, Plottable) {
+  "use strict";
+
+  d3.csv("../overlaying/tests/hockey.csv").get(function(error, rows) {
+  var data = rows.reverse(); 
```

Comment:
Ah, i started by copying another example that used csv, and one of you had done this. It worked, so I never changed it, I guess. I'll take it out if it works without.